extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -360,6 +367,21 @@ export class AmpAnalytics extends AMP.BaseElement {
               this.addTrigger_(trigger);
             } else if (trigger['selector']) {
               // Expand the selector using variable expansion.
+              if (this.multiSelectorVisibilityOn_) {
+                if (Array.isArray(trigger['selector'])) {
+                  const selectorPromises = trigger['selector'].map(s => {
+                    return this.variableService_.expandTemplate(
```

Comment:
Ah! I forgot about this. TBH I wish we can deprecate the feature to support selector variables. Can we please not support this for multi selector?

Reply:
Ya, sounds good. I'll document in the spec